File: Models/Turnir.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Collections.Generic;
using System.Text.Json.Serialization;
using System;

namespace Models
{
    [Table("Turnir")]
    public class Turnir
    {
        // Ovde pisemo propertije
        [Key]
        public int TurnirID { get; set; }

        [MaxLength(50)]
        [Required]
        public string Naziv { get; set; }

        [Required]
        public Klub Klub_organizator { get; set; }

        [Required]
        public DateTime Datum_pocetka { get; set; }

        [MaxLength(50)]
        [Required]
        public string Mesto { get; set; }

        [MaxLength(50)]
        [Required]
        public int Nagrada { get; set; }

        [JsonIgnore]
        public List<Igrac> Prijavljeni_igraci { get; set; }

        [JsonIgnore]
        public List<Igrac> Ostali_igraci { get; set; }

        [JsonIgnore]
        public List<Mec> Mecevi { get; set; }

        public Igrac Pobednik { get; set; }
        
        public Sudija Sudija { get; set; }
    }
}

File: Client/Turnir.js
export class Turnir {

    constructor(naziv, mesto, datum_poc, nagrada, klub, pobednik, sudija) {
        this.naziv = naziv;
        this.klub = klub;
        this.datum_pocetka = new Date(datum_poc);
        this.datum_pocetka = this.datum_pocetka.toLocaleDateString('en-UK');

        this.mesto = mesto;
        this.nagrada = nagrada;
        this.pobednik = pobednik;
        this.sudija = sudija;

        this.kolo = 1;

        this.prijavljeni_Igraci = [];
        this.Mecevi = [];
        this.ostali_Igraci = [];
    }

    crtaj(host) {
        var tr = document.createElement("tr");
        tr.className = "RedTurnir";
        host.appendChild(tr);

        var el = document.createElement("td");
        el.className = "NazivTurnira";
        el.innerHTML = this.naziv;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.datum_pocetka;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.mesto;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.nagrada;
        tr.appendChild(el);

        var el = document.createElement("td");
        el.innerHTML = this.klub.naziv;
        tr.appendChild(el);

        var el = document.createElement("td");
        if (this.pobednik != null) {
            el.innerHTML = this.pobednik.ime + " " + this.pobednik.prezime;
        } else
            el.innerHTML = "";
        tr.appendChild(el);

        var el = document.createElement("td");
        if (this.sudija != null) {
            el.innerHTML = this.sudija.ime + " " + this.sudija.prezime;
        } else
            el.innerHTML = "";
        tr.appendChild(el);
    }
}